catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/GeoPdfParserImpl.java

```diff
@@ -184,30 +183,33 @@ private String point2dToWkt(Point2D point2D) {
    *
    * @param lgidict - The PDF's LGIDict object
    * @param neatLineArray - The NeatLine array of points for the PDF
-   * @param toDoubleVisitor - A visitor that converts PDF Strings / Ints / Longs into doubles.
    * @return the generated WKT Lat/Lon set
    * @throws IOException
    */
-  private Optional<String> getWktFromNeatLine(
-      COSDictionary lgidict, COSArray neatLineArray, ICOSVisitor toDoubleVisitor)
+  private Optional<String> getWktFromNeatLine(COSDictionary lgidict, COSArray neatLineArray)
       throws IOException {
+    ToDoubleVisitor toDoubleVisitor;
```

Comment:
✏️ No real reason to share this reference between loops. Consider defining it locally within each loop to reduce its scope.